firmware/src/ccard.c

```diff
@@ -305,7 +305,7 @@ void CCARD_Tasks ( void )
                 TES_relay_clear(); // clears relay drive
             }
             relay_busy = false;  // relay done
-            relay = relay & ((1 << (data_bits-1))-1);  // clear relay busy bit
+            //relay = relay & ((1 << (data_bits-1))-1);  // clear relay busy bit
```

Comment:
@slacrherbst I'm not experienced with smurfc yet, but shouldn't the busy bit be cleared?